cpp/src/gandiva/gdv_function_stubs_test.cc

```diff
@@ -949,4 +949,65 @@ TEST(TestGdvFnStubs, TestMaskLastN) {
   EXPECT_EQ(expected, std::string(result, out_len));
 }
 
+TEST(TestGdvFnStubs, TestConv) {
+  gandiva::ExecutionContext ctx;
+
+  int64_t ctx_ptr = reinterpret_cast<int64_t>(&ctx);
+  gdv_int32 out_len = 0;
+
+  const char* value = conv_utf8_int32_int32(ctx_ptr, "1000101101", 10, 2, 10, &out_len);
+  std::string out_value = std::string(value, out_len);
+  EXPECT_EQ(out_value, "557");
+
+  value = conv_utf8_int32_int32(ctx_ptr, "ffa", 3, 16, 10, &out_len);
```

Comment:
Another test is that if the user defines an invalid strings:
```
123-4565
aaaa@#aa
ffe sdkl
```

Reply:
Haved some problems with this entryes, but I did modifies and add new tests for this.